packages/puppeteer-core/src/common/util.ts

```diff
@@ -313,7 +313,7 @@ export const UTILITY_WORLD_NAME =
 /**
  * @internal
  */
-export const SOURCE_URL_REGEX = /^[\040\t]*\/\/[@#] sourceURL=\s*(\S*?)\s*$/m;
+export const SOURCE_URL_REGEX = /^[\x20\t]*\/\/[@#] sourceURL=\s?(\S*?)\s?$/m;
```

Comment:
that is not equivalent, right? it only allows zero or 1 whitespace. I wonder if it could regress  on some inputs.

Reply:
I think using string functions might actually be somewhat common. Let's check for up to 10 whitespaces?